### PR TITLE
remove windows newlines

### PR DESCRIPTION
**Pull request by cjkeilig**

### CODE REVIEW

File: HCCGo/app/js/jobSubmissionCtrl.js [hunk 1]
```diff
@@ -146,7 +146,7 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
 
     // Separate SBATCH options from commands
     job.commands = editor.getValue();
-    var other = editor.getValue().split("\n");
+    var other = editor.getValue().split(/\r?\n/);
```
Comment (djw8605) [inline HCCGo/app/js/jobSubmissionCtrl.js]: Does this actually remove the \r? from the commands?

Comment (cjkeilig) [inline HCCGo/app/js/jobSubmissionCtrl.js]: I think so, after I submitted the job, I did `cat -A test.slurm` on crane and the new lines were all dollar signs so I assumed that this meant "\n"

Comment (djw8605) [inline HCCGo/app/js/jobSubmissionCtrl.js]: Yeah, sounds good to me.  If it successfully submitted after having `\r`'s in the file, then it removed them.  Natasha wasn't even able to submit a job with `\r`.

Comment (cjkeilig) [inline HCCGo/app/js/jobSubmissionCtrl.js]: This is only for the jobFile, they are stored in job history without the removal of the "\r", I forgot to change that.

Comment (djw8605) [inline HCCGo/app/js/jobSubmissionCtrl.js]: Oh, yeah.  Remove them everywhere!  You should really just remove them from the editor contents, then move on saving the commands everywhere.

**Pull request merged**